.github/workflows/stale.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-pr-label: stale
         exempt-pr-labels:
-          - keepalive
+          - pr/keepalive
```

Comment:
sorry, just seen this pr after merging https://github.com/ovrclk/akash/pull/1320
exempt-pr-labels expects comma separated list `exempt-pr-labels: keepalive,pr/keepalive`